omigami/config_default.yaml

```diff
@@ -1,4 +1,8 @@
-plotting:
-  urls:
-    NPclassifier: 'https://npclassifier.ucsd.edu/classify?smiles='
-    classyfire: 'https://gnps-structure.ucsd.edu/classyfire?smiles='
\ No newline at end of file
+client:
+  plotting:
+    urls:
+      NPclassifier: 'https://npclassifier.ucsd.edu/classify?smiles='
+      classyfire: 'https://gnps-structure.ucsd.edu/classyfire?smiles='
+
+  host:
+    prod: "https://app.omigami.com/"
```

Comment:
add dev host on your local config to access dev. We can leave just prod here. I have this in my config:

```
client:
  host:
    dev: https://dev.omigami.com/
    prod: https://app.omigami.com/
```